### PR TITLE
WAL-379

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: src/services/storageService.ts [hunk 1]
```diff
@@ -27,29 +27,84 @@ export const getPin = async (): Promise<string> => {
...
+  })
+  .catch((error: Error) =>
+    Promise.reject(new Error(`Unable to set value for key: ${STORAGE_PIN_KEY}. ${error.message}`))
+  )
 
-  const users: Map<string, IUser> = JSON.parse(users_value)
-  users.set(user.id, user)
 
-  return AsyncStorage.setItem(STORAGE_USERS_KEY, JSON.stringify(users))
+  // const result: string | null = await AsyncStorage.getItem(STORAGE_USERS_KEY)
```
Comment (sksadjad) [inline src/services/storageService.ts]: if we don't need these code block, it's better to delete them, and if we do need them at a later point, please make a comment

**Review by sksadjad — CHANGES_REQUESTED** [review]
looks good, I just added a small request for a commented block of code

File: src/services/storageService.ts (hunk 1 above)
Comment (BtencateSphereon) [reply src/services/storageService.ts]: i think i made the commit when you wrote your comment :P

**Review by sksadjad — APPROVED** [review]
LHTM

**Pull request merged**